yinsolidated/parser.py

```diff
@@ -478,6 +478,7 @@ def iterate_identities(self):
 
         if base is not None:
             base_elem = self._find_identity(base)
+            yield base_elem
```

Comment:
any associated tests that can be updated for this change?

Reply:
Yes. Just pushed.